_posts/2022-04-11-nwaku-update.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: post
+name:  "What's new in `nwaku`?"
+title:  "What's new in `nwaku`?"
+date:   2022-04-11 10:00:00 +0200
+author: hanno
+published: true
+permalink: /nwaku-update-2022-04
```

Comment:
Maybe /introducing-nwaku

Reply:
Done in [05b607c](https://github.com/vacp2p/vac.dev/pull/55/commits/05b607c2812a885741e7f51bc0b465aff978a26e)